packages/shared/src/index.ts

```diff
@@ -1,3 +1,11 @@
 export * from './config';
 export * from './types';
 export * from './utils';
+export { UserDTO,
```

Comment:
This is the related change; this should work but please let me know if there's an error on this line after making this change
```suggestion
export type { UserDTO,
```